solr/solr-ref-guide/modules/deployment-guide/pages/replica-management.adoc

```diff
@@ -73,12 +73,9 @@ http://localhost:8983/solr/admin/collections?action=ADDREPLICA&collection=techpr
 
 [source,bash]
 ----
-curl -X POST http://localhost:8983/api/collections/techproducts/shards -H 'Content-Type: application/json' -d '
+curl -X POST http://localhost:8983/api/collections/techproducts/shards/shard1/replicas -H 'Content-Type: application/json' -d '
```

Comment:
[I] Of course, if the v2 API is changing, the ref-guide docs should be updated to reflect its new form.  This often involves updating the path in the v2 example (if there is one).  If no v2 API example exists, it'd be ideal (but I guess not strictly required) to add one using syntax similar to that found here.